authz/build.gradle

```diff
@@ -0,0 +1,75 @@
+plugins {
+    id "java-library"
+    id "maven-publish"
+
+    id "com.github.johnrengelman.shadow"
+    id "com.google.protobuf"
+    id "ru.vyarus.animalsniffer"
+}
+
+description = "gRPC: Authorization"
+
+evaluationDependsOn(project(':grpc-core').path)
+
+dependencies {
+    implementation project(':grpc-protobuf'),
+            project(':grpc-core')
+
+    testImplementation project(':grpc-core').sourceSets.test.output
+
+    annotationProcessor libraries.autovalue
+    compileOnly libraries.javax_annotation
+
+    testImplementation project(':grpc-testing'),
+            project(':grpc-testing-proto')
+    testImplementation (libraries.guava_testlib) {
+        exclude group: 'junit', module: 'junit'
+    }
+
+    def xdsDependency = implementation project(':grpc-xds')
+    shadow configurations.implementation.getDependencies().minus([xdsDependency])
+    shadow project(path: ':grpc-xds', configuration: 'shadow')
+
+    signature "org.codehaus.mojo.signature:java17:1.0@signature"
+}
+
+configureProtoCompilation()
```

Comment:
You don't need this, because you don't have any .proto files in this project.

Reply:
Done